portfolio/src/main/webapp/socials.html

```diff
@@ -12,5 +12,28 @@
       <a class="active" href="socials.html">socials</a>
       <a href="misc.html">misc</a>
     </div>
+    <div>
+      <div id="content">
+        <a class="image-link" href="https://www.linkedin.com/in/aaronlopes/" target="_blank">
+          <img border="0" src="images/linkedin.png">
+        </a>
+        <div class="divider"></div>
```

Comment:
This div seems to only be for styling (margin) instead of structure/semantics, which is frowned upon.

Some alternatives I would suggest:
1) Use CSS flexbox, which is designed for this scenario (showing a 1-dimensional row of items).
2) Set a margin-left on every anchor except the first. You might find https://developer.mozilla.org/en-US/docs/Web/CSS/:first-child and/or https://developer.mozilla.org/en-US/docs/Web/CSS/:not helpful.